practice/src/index.html

```diff
@@ -38,5 +38,90 @@
       </div>
     </div>
   </header>
+
+  <main class="main-section">
+    <!-- Best Place section -->
+    <section class="place-section">
+      <div class="container wrapper-place">
+        <hgroup class="place-introduction">
+          <h2 class="text place-title">Find The Best Place to Stress Release</h2>
+          <p class="text place-description">A new way to relieve your stress by going on vacation!
+          <p class="text place-description">Find the trip you want the book it immediately!</p>
+        </hgroup>
+        <div class="tab-nav">    
+          <ul class="tab-list">
+            <li class="tab-item">
+              <a href="javascript:void(0)" class="tab-link active">Hiking</a>
+            </li>
+            <li class="tab-item">
+              <a href="javascript:void(0)" class="tab-link">Travel</a>
+            </li>
+          </ul>
+          <form class="tab-pane">
+            <div class="pane-mountain">
+              <div class="pane-content">
+                <svg width="25" height="25" viewBox="0 0 25 25" fill="none" xmlns="http://www.w3.org/2000/svg" class="icon-address">
```

Comment:
Use svg sprite. Let me check on window to make sure that why it doesn't work, because it works on my machine